tests/unit/models/package-info-cache-test.js

```diff
@@ -203,7 +203,7 @@ describe('models/package-info-cache.js', function() {
         project = fixturifyProject.buildProjectModel(Project);
         project.discoverAddons();
         pic = project.packageInfoCache;
-        projectPackageInfo = pic.getEntry(`${fixturifyProject.root}/simple-ember-app`);
+        projectPackageInfo = pic.getEntry(`${fixturifyProject.root}${path.separator}simple-ember-app`);
```

Comment:
we should probably use `path.join(fixturifyProject.root, 'simple-ember-app')` instead